### ISSUE TITLE
Cannot receive a message from previously blocked and now unblocked contact

### ISSUE THREAD

**Issue by elina2015** [status-im/status-desktop#5261]
# Bug Report

## Description

Cannot receive a message from previously blocked and now unblocked contact

## Steps to reproduce
1. Launch the app
2. Sign in with a user that has at least one contact (for example, User B)
3. Open the chat and write and receive a few messages
4. Click on the contact's image (open User B profile) and click on "Block user", click "Block user" again
5. Click on the contact's image (open User B profile) and click on "Unblock user"
6. User B send some messages to 1-1 chat
7. Check the 1-1 chat with User B

#### Expected behavior
The messages sent after User B was unblocked should be received
#### Actual behavior
The messages are not sent after User B was unblocked 

### Additional Information

- Status desktop version: https://github.com/status-im/status-desktop/commit/591a5bdf34cb2ad161e6cb4e59ecaf6c039abb2e
- Operating System: macOS Monterey, 12.3 Beta (21E5227a)

**Comment by iurimatias** [comment]
duplicate of https://github.com/status-im/status-desktop/issues/5198
